src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import core.colors.ColorSupplier;
+import java.util.Random;
+    
 public class Application {
+
+    class Ball {
```

Comment:
every class should have its own file, except for some rare cases which you will learn about later on, but for now, let's stick to this